apps/sms/views/conversation/js/conversation.js

```diff
@@ -1844,8 +1843,17 @@ var ConversationView = {
       this.mainWrapper.classList.toggle('edit');
     }
 
+    this.editHeader.removeAttribute('no-font-fit');
+
     if (!this.selectionHandler) {
-      LazyLoader.load('/views/shared/js/selection_handler.js', () => {
+      LazyLoader.load([
```

Comment:
Looks like this part could be wrapped into utils. BTW it's amazing that we have 3 different edit-mode.css for an edit form...